packages/paste-website/src/pages/content/product-style-guide.mdx

```diff
@@ -156,6 +156,8 @@ Give the thought a sentence or restructure your syntax.
 
 **Don't** use periods at the end of fragments or incomplete sentences.
 
+**Don't** use periods at the end of an anchor link if it is the last line on an alert or notification
```

Comment:
Can we frame these examples using the "Do Don't" style used for components? It will ensure a common way to highlight how to do things, add examples and make some of these pieces of text easier to scan and read.

Reply:
Love this feedback, I'm keen to break this content up and illustrate with visual examples, **but** haven't yet learned how and I'm keen not to delay this update while I do so. 

Am having trouble importing the "Do Don't" component - just broke my page trying to. Will query it in the channel and look to add in a host of these examples for the next update. Sound useful?